queries/html_tags/injections.scm

```diff
@@ -55,7 +41,7 @@
         (quoted_attribute_value (attribute_value) @_javascript)))
     (raw_text) @javascript)
   (#eq? @_type "type")
-  (#eq? @_javascript "text/javascript")
+  (#any-of? @_javascript "text/javascript" "module")
```

Comment:
Can you add a test for "module" too?

Reply:
I dont think it's necessary as it's equal to text/javascript